decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -221,6 +224,22 @@ def install
 
       private
 
+      def gem_modifier
+        @gem_modifier ||= if options[:path]
+                            "path: \"#{options[:path]}\""
+                          elsif branch.present?
+                            "git: \"https://github.com/decidim/decidim.git\", branch: \"#{branch}\""
+                          else
+                            "\"#{Decidim::Generators.version}\""
+                          end
+      end
+
+      def branch
+        return if options[:path]
+
+        @branch ||= options[:edge] ? "chore/rails-6-upgrade" : options[:branch].presence
```

Comment:
I understand we'll need to update this right after this PR is merged, right?

Reply:
Yes, after merging the branch should be reverted to "develop"